app/assets/stylesheets/was_seed.scss

```diff
@@ -32,6 +29,7 @@
     @include shadow(2);
     background-color: $white-color;
   }
+  margin: 10px;
```

Comment:
So we are already specifying part of the margin in line 22. Can we not also add marin-left and margin-right here? Or reduce it to 5px? I think the gaps are too large here.